pkgs/top-level/all-packages.nix

```
@@ -756,6 +756,8 @@ in
 
   git-fire = callPackage ../tools/misc/git-fire { };
 
+  git-repo-updater = callPackage ../development/tools/git-repo-updater { };
```

```suggestion
  git-repo-updater = python37Packages.callPackage ../development/tools/git-repo-updater { };
```

Then you directly have all of the Python packages in scope.

---

I didn’t know you could do this. Good idea!